sound/soc/sof/intel/hda.c

```diff
@@ -713,6 +724,21 @@ int hda_dsp_probe(struct snd_sof_dev *sdev)
 	/* set default mailbox offset for FW ready message */
 	sdev->dsp_box.offset = HDA_DSP_MBOX_UPLINK_OFFSET;
 
+	/* need to power-up core before setting-up capabilities */
+	ret = hda_dsp_core_power_up(sdev, HDA_DSP_CORE_MASK(0));
```

Comment:
@plbossart What is the logic after probe completes? It seems we leave the core powered up,

Reply:
no. pm_runtime kicks in and we suspend.